webpack.config.js

```diff
@@ -1,3 +1,4 @@
+"use strict";
```

Comment:
this is required for old webpack. It's a good way to have this. the errors are not super friendly and it never say the package is out of date.